pages/docs/tutorials/native/apple-framework.md

```diff
@@ -70,28 +69,29 @@ We need to have a Kotlin/Native compiler on our machines.
 You may have a look at the
 [A Basic Kotlin/Native Application](basic-kotlin-native-app.html#obtaining-the-compiler)
 tutorial for more information on performing this step.
-Let's assume we have a console, where `kotlinc` command is available. 
+Let's assume we have a console, where the `kotlinc` command is available. 
 
-Now we call the following commands to compile the code into Frameworks
-for macOS, iOS and iOS emulator respectively:
+Now let's call the following commands to compile the code into Frameworks
+for macOS, iOS, and iOS emulator respectively:
 ```bash
 kotlinc lib.kt -produce framework -target macos_x64 -output macOS/Demo
 kotlinc lib.kt -produce framework -target ios_arm64 -output iOS/Demo
 kotlinc lib.kt -produce framework -target ios_x64 -output iOS_emu/Demo
 ```
 
-The `kotlinc` generates three frameworks for us, named `Demo.framework` under 
-`macOS`, `iOS` and `iOS_emu` folders respectively.
+The `kotlinc` generates three Frameworks for us, named `Demo.framework` under 
```

Comment:
What's the purpose of using capital "F" in "Framework"?

Reply:
Fixed